ci/drone/functions.star

```diff
@@ -14,15 +14,16 @@ def download_from_boostCI(filename, boostCI_dir, out_dir=None):
     out_dir = boostCI_dir
   url = 'https://github.com/boostorg/boost-ci/raw/master/%s/%s' % (boostCI_dir, filename)
   target_path = '%s/%s' % (out_dir, filename)
-  return 'echo "Downloading {0} to {1}"; curl -s -S --retry 10 --create-dirs -L "{0}" -o "{1}" && chmod 755 {1}'.format(url, target_path)
+  # return 'echo "Downloading {0} to {1}"; curl -s -S --retry 10 --create-dirs -L "{0}" -o "{1}" && chmod 755 {1}'.format(url, target_path)
+  return 'curl -s -S --retry 10 --create-dirs -L "{0}" -o "{1}" && chmod 755 {1}'.format(url, target_path)
```

Comment:
Then maybe remove the silent option from `curl`?

Reply:
Not sure where `set -x` is being set, but looking at log output in jobs from drone.cpp.al, it shows the full curl command including the file being downloaded. Maybe that provides enough info. It prints this:
```
curl -s -S --retry 10 --create-dirs -L "https://github.com/boostorg/boost-ci/raw/master/ci/drone/linux-cxx-install.sh" -o "ci/drone/linux-cxx-install.sh" && chmod 755 ci/drone/linux-cxx-install.sh
```